.gitlab/custom-jobs-and-variables.yml

```diff
@@ -19,7 +19,7 @@ variables:
 # Note: We repeat the reservation, necessary when jobs are manually re-triggered.
   RUBY_JOB_ALLOC: "--overlap --reservation=ci --nodes=1"
 # Project specific variants for ruby
-  PROJECT_RUBY_VARIANTS: ""
+  PROJECT_RUBY_VARIANTS: "+tests"
```

Comment:
It might be worth enabling the tests in a separate branch.

Reply:
@adayton1 I can confirm that by switching off tests I get the CI to pass.